app/models/manageiq/providers/physical_infra_manager.rb

```diff
@@ -153,10 +153,6 @@ def console_supported?
       false
```

Comment:
@Kuldip-Nanda I think we should delete the `console_supported?` method from here and also delete the `supports :console` block above since this logic should move to the BaseManager

Reply:
We should then add a `supports_not :native_console` to the base manager, then the providers which do support this can add `supports :native_console`.